shared/common-adapters/clickable-box.desktop.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import React from 'react'
+import FlatButton from 'material-ui/FlatButton'
+import Box from './box'
+
+import type {Props} from './clickable-box'
+
+export default function ClickableBox (props: Props & {children: any}) {
+  return (
+    <FlatButton {...props} style={{textAlign: 'left', height: undefined, width: '100%', display: 'flex', flexDirection: 'column', alignItems: 'stretch'}}>
```

Comment:
This is to make the underlying button element in the dom follow what the div usually does in box, which is to fill its available width.

Without it, it doesn't fill the it's whole available width like so:
<img width="641" alt="screen shot 2016-06-30 at 10 25 43 am" src="https://cloud.githubusercontent.com/assets/594035/16497726/4ee11910-3ead-11e6-9de9-5dd0bc54edb0.png">


Here's what it would look like w/o `width: 100%`
<img width="681" alt="screen shot 2016-06-30 at 10 24 58 am" src="https://cloud.githubusercontent.com/assets/594035/16497702/2c0519d2-3ead-11e6-9262-61db4629c5ae.png">

With it: (matches what it looked like with boxes)
<img width="683" alt="screen shot 2016-06-30 at 10 29 14 am" src="https://cloud.githubusercontent.com/assets/594035/16497781/8a09ad40-3ead-11e6-8480-3ff7f1961e07.png">